app/dashboard/templates/profiles/scorecard_funder.html

```diff
@@ -0,0 +1,35 @@
+{% load i18n static avatar_tags %}
+  <div class="col-12 col-lg-6">
+    <div class="profile-header__stats profile-header__stats--funder card mb-4">
+      <div class="card-header font-subheader">
+        {% if scoreboard_position_funder %}
+          <b>#{{ scoreboard_position_funder }}</b>
+        {% else %}
+          <b>&nbsp;</b>
+        {% endif %}
+        {% trans "Funder" %}
+      {% include "shared/profile_rank_link.html" with link='payers' %}
+      </div>
+      <div class="card-body font-subheader">
+        <ul>
+          <li><b>{{ funded_bounties_count }}</b> {% trans "bounties funded" %}</li>
+          <li><b>{{ sum_eth_funded|floatformat:2 }} ETH</b> {% trans "funded" %}</li>
+          <li class="works_with_list">
+            {% if works_with_funded|length %}
+              <span class="font-body"><i class="fab fa-ethereum"></i> {% trans "Funds" %}:</span>
+              {% for profile, num_times in works_with_funded.items %}
+                {% if forloop.counter < 6 %}
```

Comment:
can use a template tag here `|slice:":5"` instead of the if check

Reply:
hmm... this is code that i didnt write (i just moved location).. will take a look tho